src/main/java/com/jalasoft/jfc/model/video/VideoConverter.java

```diff
@@ -52,24 +52,24 @@
      * @throws IOException
      */
     public FileResult convert(Param param) {
-
+        System.out.println("convert ---");
         VideoParam videoParam = (VideoParam) param;
         FileResult fileResult = new FileResult();
 
         try {
             List<ICommandStrategy> list = new ArrayList<>();
             list.add(new CommandFFMpegPath());
-            list.add(new CommandFFMpeg());
             list.add(new CommandInputFilePath(videoParam.getInputPathFile()));
-            list.add(new CommandVideoAspectRatio(Integer.toString(videoParam.getAspectRatio())));
+            list.add(new CommandVideoAspectRatio(videoParam.getAspectRatio()));
             list.add(new CommandVideoScale(videoParam.getWidth(), videoParam.getHeight()));
-            list.add(new CommandVideoConverter());
-            list.add(new CommandVideoThumbNail(Integer.parseInt(videoParam.getThumbnail())));
+            //list.add(new CommandVideoConverter());
+            list.add(new CommandVideoThumbNail("00:00:2.100"));
```

Comment:
if the thumbnail value is that string, you can add inside of the thmbnail class like a final variable